packages/@okta/vuepress-theme-prose/assets/css/layouts/_layout.scss

```diff
@@ -4,6 +4,7 @@ body {
   font-weight: 400;
   line-height: 1.5;
   font-feature-settings: "lnum","pnum";
+  background-color: #16262f;
```

Comment:
I see this color use multiple times, can we create a special variable for it?

Reply:
👍 , just added da52d5f to address this